pkg/types/types.go

```diff
@@ -193,6 +193,7 @@ type ResourcePool interface {
 	GetMounts(deviceIDs []string) []*pluginapi.Mount
 	StoreDeviceInfoFile(resourceNamePrefix string) error
 	CleanDeviceInfoFile(resourceNamePrefix string) error
+	GetCDIKind() string
```

Comment:
this should be GetCDIName() string

since this bit is essentially the name segment of Kind. see: [1]

[1] https://github.com/cncf-tags/container-device-interface/blob/main/SPEC.md#kind

Reply:
done